### ISSUE TITLE
The 'width' and 'height' in the manifest file does not work properly on OSX.[nw 0.13 rc1]

### ISSUE THREAD

**Issue by sean220** [nwjs/nw.js#4493]
The behavior of these two parameters for main window is unstable on OSX,Sometimes it loads the old value, or get some unpredictable values.

**Comment by yeaphone** [comment]
I meet the same problem.

**Comment by rogerwang** [comment]
Did it restore with the dimension when you close it last time?

**Comment by sean220** [comment]
YES,After tests found it's just restores the last dimension and Ignores the value in the package.json even updated.

**Comment by JaredSartin** [comment]
fullscreen doesn't take either.
